test/integration/smoke/test_network.py

```diff
@@ -2043,4 +2060,341 @@ def test_03_destroySharedNetwork(self):
             len(nic_ip_address),
             0,
             "Failed to find the placeholder IP"
-        )
\ No newline at end of file
+        )
+
+
+class TestNetworkWithMtuConfiguration(cloudstackTestCase):
```

Comment:
Just my opinion, this can be a class in a separate file and maybe a component test. Smoke tests already take over 12 hours. Also in case of failure for any test in this file all the tests will run again.

Reply:
Sure , with will do